src/Features/Core/Portable/Diagnostics/DiagnosticAnalyzerExtensions.cs

```diff
@@ -16,7 +16,6 @@ internal static class DiagnosticAnalyzerExtensions
     public static bool IsWorkspaceDiagnosticAnalyzer(this DiagnosticAnalyzer analyzer)
         => analyzer is DocumentDiagnosticAnalyzer
         || analyzer is ProjectDiagnosticAnalyzer
-        || analyzer == FileContentLoadAnalyzer.Instance
```

Comment:
FileContentLoadAnalyzer *is* just a DocumentDiagnosticAnalyzer now.  so we don't need this special check.